Add Documentation after 0.3.0 release (#94)
AbsaOSS/hermes: _docs/0.3/usage/plugins/plugin-creation.md

---
layout: docs
title: Usage - Plugin Creation
version: '0.3'
categories:
    - '0.3'
    - 'usage'
    - 'plugins'
redirect_from: /docs/usage/plugins/plugin-creation
---

> E2E Runner is also able to run any plugin on class path. The plugin needs to extend `za.co.absa.hermes.e2eRunner.Plugin` trait and the `performAction` method needs to return a `za.co.absa.hermes.e2eRunner.PluginRsult` implementation.

Reviewer: ```suggestion
E2E Runner is also able to run any plugin on the classpath. The plugin needs to extend `za.co.absa.hermes.e2eRunner.Plugin` trait and the `performAction` method must to return an implementation of `za.co.absa.hermes.e2eRunner.PluginResult`.
```

redirect_from: /docs/usage/plugins/plugin-creation
---

> E2E Runner is also able to run any plugin on the classpath. The plugin needs to extend `za.co.absa.hermes.e2eRunner.Plugin` trait and the `performAction` method must to return an implementation of `za.co.absa.hermes.e2eRunner.PluginResult`.

Reviewer: Maybe decide if you want to write `class path` or `classpath`  and then unify throughout the documentation? Both variants seem to be used now.
